### PR TITLE
Change spinner to be an in button spinner

### PR DESCRIPTION
**Pull request by filg68**

### CODE REVIEW

**Comment by filg68** [comment]
I added the spinner to Poll Create and Edit Dialog save button.  In the Edit Dialog the save happens so fast that the button does not update completely and you get this weird size increase and never see the spinner.  If the save took longer I guess it would be ok but as is at least on my machine it doesn't look that good.

File: client/src/components/AdornedButton.js [hunk 1]
```diff
@@ -5,15 +5,21 @@ import Button from "@material-ui/core/Button";
...
 
-const styles = {
+const styles = theme => ({
     root: {
         marginLeft: 5
+    },
+    spinner: {
+        position: "relative",
+        marginLeft: "7px",
+        marginRight: "5px",
```
Comment (IoPhoenix) [inline client/src/components/AdornedButton.js]: Could you please remove margin-right, it makes right side of the button with the spinner look wider

File: client/src/components/EditProfileDialog.js [hunk 2]
```diff
@@ -63,6 +63,11 @@ const styles = theme => ({
             textTransform: "initial"
         }
     },
+    loadingButton: {
+        color: theme.palette.grey[500],
+        disabled: true,
+        padding: "4px 4px 4px 22px"
```
Comment (IoPhoenix) [inline client/src/components/EditProfileDialog.js]: Do you remember we talked about buttons getting their padding from theme.js? :) You can remove this custom padding and button will look the same

File: client/src/components/EditProfileDialog.js [hunk 3]
```diff
@@ -63,6 +63,11 @@ const styles = theme => ({
             textTransform: "initial"
         }
     },
+    loadingButton: {
+        color: theme.palette.grey[500],
+        disabled: true,
```
Comment (IoPhoenix) [inline client/src/components/EditProfileDialog.js]: I don't think this is valid css :(

File: client/src/components/AdornedButton.js [hunk 4]
```diff
@@ -5,15 +5,21 @@ import Button from "@material-ui/core/Button";
...
+        color: theme.palette.secondary.main
     }
-};
+});
 
-const SpinnerAdornment = withStyles(styles)(props => (
-    <CircularProgress className={props.classes.spinner} size={18} />
-));
+const SpinnerAdornment = withStyles(styles)(props => {
+    return <CircularProgress className={props.classes.spinner} size={18} />;
```
Comment (IoPhoenix) [inline client/src/components/AdornedButton.js]: The spinner class is now being applied, but root class not, could you please remove it completely then?

File: client/src/components/AddPollDialog.js [hunk 5]
```diff
@@ -200,7 +203,6 @@ class AddPollDialog extends Component {
             <div>
                 <Button
                     aria-label="create poll button"
-                    className={classes.button}
```
Comment (IoPhoenix) [inline client/src/components/AddPollDialog.js]: Can you please remove button styles from styles object as they are redundant now

**Pull request merged**